app/elements/leaflet-wrapper/leaflet-wrapper.html

```diff
@@ -22,7 +22,12 @@
     <template is="dom-if" if="{{hasMultipleLocs}}">
       <leaflet-map fit-to-markers>
         <template is="dom-repeat" items="{{mapData}}" as="loc">
-          <leaflet-marker latitude="{{loc.latitude}}" longitude="{{loc.longitude}}"></leaflet-marker> 
+          <leaflet-marker latitude="{{loc.latitude}}" longitude="{{loc.longitude}}" title="{{loc.city}}, {{loc.state}}:{{loc.count}}"></leaflet-marker> 
+            <!-- <leaflet-circle longitude="{{loc.longitude}}" latitude="{{loc.latitude}}"
```

Comment:
OK.  In the future, please include a comment with an explanation, else others might think it's stale/unneeded code.